README.md

```diff
@@ -34,3 +34,42 @@ Shuttle Tracker needs configuration to run properly. The preferred method during
 ### Environment variables
 
 Most keys can be overridden with environment variables. The variables names usually take the format `SECTION_KEY`. For example, overriding database's Mongo URL could be done with a variable named `POSTGRES_URL`.
+
+
+## Setting up (Windows)
+
```

Comment:
I installed Go (unsuccessfully) a few months ago, but I didn't uninstall the older version before following these steps, which caused some issues. For example, my `GOPATH` was set correctly, but the folder `C:/Users/rolleg/go` did not exist. I could also run `go get ...`, but it wasn't working or generating any sort of error message.

I would add a step to uninstall any unsuccessful Go install before starting and provide [this link](https://golang.org/doc/install#uninstall).